components/Card.js

```diff
@@ -5,18 +5,17 @@ const CardImage = styled(Image)`
   border-radius: 10px;
   margin: 10px;
   box-shadow: 0px 3px 10px rgba(0, 0, 0, 0.3);
-  width: 180px;
-  max-width: 250px;
+  width: 100%;
+  min-width: 180px;
+  max-width: 180px;
```

Comment:
now you could use only width: 180px since nothing else is possible. If you really want to fix the width of the image to that given size, then please just use that. Otherwise you need to remove either min or max or give them different values.